GetMessage/handler.ts

```diff
@@ -435,6 +450,7 @@ export const GetMessageHandler = (
  */
 // eslint-disable-next-line max-params, prefer-arrow/prefer-arrow-functions
 export function GetMessage(
+  { FF_PAYMENT_STATUS_ENABLED }: IConfig,
```

Comment:
Why destructuring object here instead of passing a simple boolean?

Reply:
Just in case we'll need more values in future, nothing else actually.. What do you prefer?

Reply:
Since we have to remove it once we have a stable backend service, It's ok :D

Reply:
Right :D